examples/cc_gwlb/variables.tf

```diff
@@ -221,6 +221,12 @@ variable "allowed_principals" {
   default     = []
 }
 
+variable "deregistration_delay" {
+  type        = number
+  description = "Amount time for Elastic Load Balancing to wait before changing the state of a deregistering target from draining to unused. The range is 0-3600 seconds."
+  default     = 0
```

Comment:
I believe you told me about this but I can't remember.  default was 300.  making it 0 means we are not giving any time for existing flows sent to be processed and delivered back.